walkthroughs/programming-syntax.md

```diff
@@ -34,11 +34,11 @@ If this is your first time developing an Instant Answer, check out our [detailed
 
 ## Create the Instant Answer Page
 
-> If you're just using this walkthrough to learn and practice for now, you can skip this step. This is important only if you plan to submit what you're working on.
+> If you're just using this walkthrough to learn and practice for now, you can skip this step. This is important only if you plan to submit what you're working on. 
 
 Every Instant Answer on DuckDuckGo.com has its very own [*Instant Answer Page*](https://duck.co/ia/). These are the home base for planning, collaboration, and metadata. Instant Answer pages also show any Github issues and let you know what stage the Instant Answer is in.
 
-- If you're building a brand new cheat sheet, start by [creating a new Instant Answer page](https://duck.co/ia/new_ia).
+- If you're building a brand new cheat sheet, start by [creating a new Instant Answer page](https://duck.co/ia/new_ia). Please remember to end the name of the Instant Answer with `_cheat_sheet` so that the id in the json file and the page are same.
```

Comment:
Should this also mention capitalization rules?

Reply:
Yeah! I'll add that too